stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -103,7 +103,7 @@ public final class _DataStorage {
     public var _length: Int
     public var _capacity: Int
     public var _needToZero: Bool
-    public var _deallocator: ((UnsafeMutableRawPointer, Int) -> Void)? = nil
+    public var _deallocator: ((UnsafeMutableRawPointer, Int) -> Void)?
```

Comment:
@jrose-apple Whoops, my bad. I've now updated my gardening scripts not to hunt for redundant initializations to `nil` :-)  Thanks for notifying.